### DISCUSSION TITLE
JSON 转 Go Struct的功能字段命名风格优化

### DISCUSSION THREAD

**Discussion by zihaojin** [he3-app/he3-feedback#43] [Ideas]
这个功能生成的字段名称下划线命名法。golang通常的变量风格应该是驼峰式。

**Comment by hongchaodeng** [comment]
你好，感谢你提的问题。

我查看 He3 后，现在是 CamelCase 的：

<img width="968" alt="Snipaste_2023-03-08_18-38-19" src="https://user-images.githubusercontent.com/920884/223691194-2a444af8-432e-467e-a9eb-b6b76a333f5e.png">

**Reply by zihaojin** [reply]
![image](https://user-images.githubusercontent.com/24381604/223699348-32014e2b-4305-4a3b-84fe-1b780006c3cb.png)
哦，看起来和json中原本的字段风格有关系，转化是和json中原有的风格一致的。json一般都是下划线风格的。

**Reply by hongchaodeng** [reply]
明白了。非常感谢你提供的例子！我们会尽快修复
